docs/cloud/pouta/persistent-volumes.md

```diff
@@ -154,6 +154,8 @@ the virtual machine to avoid data loss!**
 openstack server remove volume <server> <volume>
 ```
 
+When an instance with an attached volume is [shelved](vm-lifecycle.md/#shelved), the volume status changes to **Reserved**. Unshelving the instance changes it back to **In-use**, and deleting the instance sets it to **Available**. For **multiattach volumes**, the status becomes **Reserved** when all attached instances are shelved, preventing the volume from being attached to another instance. It returns to **In-use** as soon as at least one of the attached instances is **unshelved**.
```

Comment:
```suggestion
- When an instance with an attached volume is [shelved](vm-lifecycle.md/#shelved), the volume status changes to **Reserved**. Unshelving the instance changes it back to **In-use**, and deleting the instance sets it to **Available**.
- For **multiattach volumes**, the status becomes **Reserved** when all attached instances are shelved, preventing the volume from being attached to another instance. It returns to **In-use** as soon as at least one of the attached instances is **unshelved**.
```

Reply:
I think it improves readability to separate both cases (multiattach and not).

Reply:
Yes, it is good to separate.